httpcore5-testing/src/test/java/org/apache/hc/core5/testing/nio/H2CoreTransportTest.java

```diff
@@ -55,11 +55,15 @@ abstract class H2CoreTransportTest extends HttpCoreTransportTest {
     private final H2AsyncRequesterResource clientResource;
 
     public H2CoreTransportTest(final URIScheme scheme) {
+        this(scheme, null);
+    }
+
+    public H2CoreTransportTest(final URIScheme scheme, final String protocol) {
```

Comment:
@reta Can we give the variable a  better name to make it clearer what protocol it is? `tlsVersion` or `tlsProtocol` maybe?